workspace-manager-clienttests/src/main/resources/servers/workspace-resiliency.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "workspace-resiliency",
```

Comment:
This name doesn't make sense to me. Resilience tests are not specific to a server, all servers should be able to run them. I think it should be `workspace-ichang` instead, to follow the pattern of existing server specifications.

Reply:
Updated the name of the config.